### ISSUE TITLE
[images, not chroot] Arch: failed to fetch growpart

### ISSUE THREAD

**Issue by jonhoo** [hartwork/image-bootstrap#73]
```
Fetching growpart of cloud-utils...
# wget -O/mnt/tmprnmZ1O/usr/bin/growpart https://bazaar.launchpad.net/~cloud-utils-dev/cloud-utils/trunk/download/head:/growpart-20110225134600-d84xgz6209r194ob-1/growpart
--2019-01-14 14:01:55--  https://bazaar.launchpad.net/~cloud-utils-dev/cloud-utils/trunk/download/head:/growpart-20110225134600-d84xgz6209r194ob-1/growpart
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving bazaar.launchpad.net (bazaar.launchpad.net)... 91.189.95.84
Connecting to bazaar.launchpad.net (bazaar.launchpad.net)|91.189.95.84|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-01-14 14:01:56 ERROR 404: Not Found.

...

Error: Command "wget -O/mnt/tmprnmZ1O/usr/bin/growpart https://bazaar.launchpad.net/~cloud-utils-dev/cloud-utils/trunk/download/head:/growpart-20110225134600-d84xgz6209r194ob-1/growpart" returned non-zero exit status 8
```

**Comment by hartwork** [comment]
I see, thanks for the report! Let's find some promising replacement URL with SSL and use that for a fix. First person with a candidate wins, go! :smile:

**Comment by hartwork** [comment]
Fixed by c385ded39d6de0b21655c74a981427ad33bf30f5, closing.
